utils/cardInfoUtil.ts

```diff
@@ -0,0 +1,89 @@
+import { ProgramCardInfo } from "models/Program";
+import { ClassCardInfo } from "models/Class";
+
+export class CardInfoUtil {
+    static getClassCardInfo(findResults: any[]): ClassCardInfo[] {
```

Comment:
Let's make this plural `getClassCardInfos` maybe? I think having it plural would make it more reflective since we're passing and returning an array.

Reply:
Let's add some documentation too please!